obelisk/client.py

```diff
@@ -64,9 +64,12 @@ class LibbitcoinClient(ClientBase):
         'validate'
     ]
 
-    def __init__(self, address, public_key=None, log=None, heartbeat_port=9092):
+    def __init__(self, address, addresses=None, public_key=None, log=None, heartbeat_port=9092):
```

Comment:
I switched it over to allow address to be an array. I also randomized the server list over here:
https://github.com/OpenBazaar/OpenBazaar-Server/pull/303/commits/7cd19e00beb3ee7aaffd3ecc8f08345a6be06692

For geo ip targeting what service should we use? freegeoip.net looks good to me.